lambdas/service/app.py

```diff
@@ -448,15 +448,16 @@ def vendor_html(*path: str) -> str:
 def swagger_ui():
     swagger_ui_template = vendor_html('swagger-ui.html.template.mustache')
     swagger_ui_html = chevron.render(swagger_ui_template, {
-        'OAUTH2_CLIENT_ID': config.google_oauth2_client_id,
+        'OAUTH2_CLIENT_ID': ('null' if config.google_oauth2_client_id is None
+                             else repr(config.google_oauth2_client_id)),
```

Comment:
repr only works coincidentally. JSON is a subset of JS syntax so you should use that. Then you don't need to hard-code `null` but can use `None`. The conditional goes away, too I think.